source/common/runtime/runtime_features.cc

```diff
@@ -45,6 +45,41 @@ constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
     "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
+    // 1.12.0
+    "envoy.deprecated_features.route.proto:regex_match",
+    "envoy.deprecated_features.route.proto:regex",
+    "envoy.deprecated_features.accesslog.proto:config",
+    "envoy.deprecated_features.thrift_proxy.proto:config",
+    "envoy.deprecated_features.cds.proto:tls_context",
```

Comment:
Same, just done, seems not good.

Reply:
Given this has been stale for a month, WDYT? :-P

Reply:
My concern is still that this is a config option that almost everyone sets. With that said, after a month, it's probably fine. @lizan WDYT?